app/browser/api/ledger.js

```diff
@@ -2291,6 +2291,16 @@ const fetchReferralHeaders = () => {
   }, {}, appActions.onFetchReferralHeaders)
 }
 
+const scheduleFetchReferralHeaders = () => {
+  const startupFetchDelay = appConfig.ledger.startupFetchDelay
+  // Delay should not occur after client boot has occured
+  const delay = (startupFetchDelay && !bootP) ? startupFetchDelay : 0
+
+  setTimeout(() => {
```

Comment:
We need to track timeout and clear it before we add a new one

Reply:
++